vividus-http-client/src/main/resources/org/vividus/http/client/spring.xml

```diff
@@ -28,8 +28,12 @@
     </bean>
 
     <bean id="httpClientConfig" class="org.vividus.http.client.HttpClientConfig">
-        <property name="sslCertificateCheckEnabled" value="${http.ssl.check-certificate}" />
-        <property name="sslHostnameVerificationEnabled" value="${http.ssl.verify-hostname}" />
+        <property name="sslConfig">
+            <bean class="org.vividus.http.client.SslConfig">
+                <property name="sslCertificateCheckEnabled" value="${http.ssl.check-certificate:true}" />
+                <property name="sslHostnameVerificationEnabled" value="${http.ssl.verify-hostname:true}" />
```

Comment:
what's the point to keep default in spring.xml and *.properties?

Reply:
was in the process of moving all properties from http client and util from xml to props, now its done